.github/workflows/check_pr_target.yml

```
@@ -0,0 +1,27 @@
+name: Check if PRs have the right target (base) branch
+
+on: pull_request
```

IIRC `edited` is not triggering by default, so if you chose the wrong target branch by mistake and subsequently change it to `dev`, the check won't re-run unless `edited` is in the trigger list
```suggestion
on: pull_request:
  types: [opened, synchronize, reopened, edited]
```